dumb-jump.el

```diff
@@ -2097,15 +2177,16 @@ to keep looking for another root."
          (result (--filter
                   (s-ends-with? (concat "." (plist-get it :ext)) filename)
                   dumb-jump-language-file-exts)))
-    (when result
+    (when (and result (eq (length result) 1))
       (plist-get (car result) :language))))
 
 (defun dumb-jump-issue-result (issue)
   "Return a result property list with the ISSUE set as :issue property symbol."
   `(:results nil :lang nil :symbol nil :ctx-type nil :file nil :root nil :issue ,(intern issue)))
 
 (defun dumb-jump-get-results (&optional prompt)
-  "Run dumb-jump-fetch-results if searcher installed, buffer is saved, and there's a symbol under point."
+  "Run dumb-jump-fetch-results if searcher installed, buffer is saved,
+and there's a symbol under point."
```

Comment:
The first line of a docstring should be a complete sentence (checkdoc).

Reply:
I actually missed this in a few of the last docstring, that should be fixed too.